Neos.Neos/Classes/Fusion/Cache/ContentCacheFlusher.php

```diff
@@ -64,6 +65,8 @@ public function __construct(
     /**
      * Main entry point to *directly* flush the caches of a given NodeAggregate
      *
+     * FIXME workspaceName instead of contentStreamId
+     *
```

Comment:
```suggestion
```

this is too far up, i would miss deleting this 😅

Reply:
It's about the param? Where else should it go without destroying the param block

Reply:
idk ill take care of this, dont worry about having to annotate it all ;)